view_index.php

```diff
@@ -0,0 +1,39 @@
+<?php
+require_once('controller.php');
+
+$todos = new Controller('post_todo');
+$todos->index();
+
+?>
+
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>todo</title>
+</head>
+<body>
+<a href="view_new.php">新規作成</a>
+    <table>
+        <tr>
+            <th>タイトル</th>
+            <th>内容</th>
+            <th>投稿日時</th>
+            <th>更新日時</th>
+        </tr>
+        <?php foreach($todos->index() as $column){ ?>
+        <tr>
+            <td><a href="/php_todoapp/view_update.php?id=<?php if(isset($column)){echo $column['id'];} ?>">
+                <?php if(isset($column)){echo $column['title'];} ?></a></td>
```

Comment:
存在確認する意識とてもいいともいます！！
ただ、foreachの場合存在しなかったらloopに入らないと思うので、確認するなら、titleのkeyがなかったらechoしないとかの方が親切かなと思います。

多分php8を使っているかと思うので、こんな書き方もできますよ。

```suggestion
                <?php echo $column['title'] ?? '' ?></a></td>
```